modules/displayPokemon.js

```diff
@@ -0,0 +1,78 @@
+import { getComments } from './involvementRequests.js';
+import displayMessage from './displayMessage.js';
+/* eslint-disable no-new */
```

Comment:
Please don't disable the linters. Consider fixing the linter issues instead of disabling the linters.